app/controllers/projects/samples/attachments/deletions_controller.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    module Attachments
+      # Controller actions for Project Samples Attachments Concatenation
+      class DeletionsController < Projects::Samples::ApplicationController
+        respond_to :turbo_stream
+
+        def new
+          authorize! @project, to: :update_sample?
+          render turbo_stream: turbo_stream.update('sample_files_modal',
+                                                   partial: 'modal',
+                                                   locals: {
+                                                     open: true
+                                                   }), status: :ok
+        end
+
+        def destroy # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
+          authorize! @project, to: :update_sample?
+
+          atts_to_delete = get_attachments(deletion_params['attachment_ids'])
+          atts_to_delete_count = atts_to_delete.count
+
+          atts_to_delete.each do |attachment|
+            atts_to_delete -= ::Attachments::DestroyService.new(@sample, attachment, current_user).execute
+          end
+
+          respond_to do |format|
```

Comment:
**nitpick (blocking):** Since `respond_to :turbo_stream` is defined on line 8 of this controller you do not need a `respond_to` block in this method. You only need to keep lines 32 - 41 as the surround `respond_to` and `format.turbo_stream` are redundant.

Reply:
Removed in e3fd4b79632c27b3760a275b9e00e27048567bcf.